src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Avoid importing by `*`
You may leave it as it is for now, however in 4kloc, such coding standards will be enforced, you might want to refer to the coding standards for the module.